Makefile

```diff
@@ -446,7 +446,11 @@ vendor:
 
 # Full build inside a docker container for a clean release build
 docker-build: vendor
-	docker build --build-arg VERSION=$(VERSION) -t provenance-io/blockchain . -f docker/blockchain/Dockerfile
+ifeq ($(UNAME_M),x86_64)
+	docker build --build-arg VERSION=$(VERSION) --build-arg ARCH=$(UNAME_M) -t provenance-io/blockchain . -f docker/blockchain/Dockerfile
+else
```

Comment:
this flow would allow something that isn't x86_64 or aarch64 through... but of course that would blow up immediately so I guess that isn't much of a risk